deploy/operator.yaml

```diff
@@ -33,3 +34,13 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "pravega-operator"
+            - name: SECRET_NAME
```

Comment:
why `SECRET_NAME`? I see that is being used as the webhook server name, but not sure why we can't use the operator name.

Reply:
Hmm. I think the server name is `admission-webhook-server`
So the reason for passing this as an environment variable is to let webhook knows the `secret` name that it can use. But I just find that we may skip the `secret` part, I will update the change later.